manage/groups/settings.rst

```diff
@@ -22,6 +22,9 @@ Parent group
    permissions. That means you can treat each group as an individual group -
    even if they have a child/parent relation.
 
+   Zammad supports 10 levels of hierarchy for groups on PostgreSQL systems.
+   If you are using a MySQL database, it is limited to 6 levels.
+
```

Comment:
How about adding a hint about migrating to PostgreSQL for existing systems? Like "Did you know that Zammad will deprecate MySQL with version..." with a link to appropriate section on migration. Just a thought, since we should try to communicate in a clear way that we had enough with these MySQL limitations :)